### ISSUE TITLE
[Question]: 

### ISSUE THREAD

**Issue by thewh1teagle** [playwright-community/playwright-go#494]
I would like to use system wide preinstalled webviews as the browsers for the automation.
eg. on Windows with `webview2`. on macOS with `webkit` and linux with `wkwebview` so I won't need to install heavy browser such as chromium.

Is that possible? Thanks.

**Comment by canstand** [comment]
No. However, you can use the Chrome or Edge browser already installed on your system, although it is not recommended. 

https://github.com/playwright-community/playwright-go/issues/340

https://playwright.dev/docs/browsers#google-chrome--microsoft-edge
https://playwright.dev/docs/webview2

**Comment by mxschmitt** [comment]
Native system webviews (WebView2 / WKWebView) can't be automated directly — Playwright only drives browsers that speak its protocol.

To avoid the Chromium download, you can instead use a browser already installed on the system:

```go
browser, err := pw.Chromium.Launch(playwright.BrowserTypeLaunchOptions{
    Channel: playwright.String("chrome"), // or "msedge"
})
```

Or point at any Chromium binary via `ExecutablePath`. For WebView2 specifically, launch your WebView2 app with a remote-debugging port and attach with `ConnectOverCDP`.

See https://playwright.dev/docs/browsers#google-chrome--microsoft-edge and https://playwright.dev/docs/webview2.

Closing as answered.
